src/lti_toolbox/launch_params.py

```diff
@@ -4,16 +4,26 @@
 This is based on lti library (https://github.com/pylti/lti).
```

Comment:
Should I rename this file to a more generic name, as `params.py`? I didn't as it would be a breaking change.